digdag-ui/console.jsx

```diff
@@ -815,12 +816,23 @@ function formatTaskState(state) {
   }
 }
 
+const JobLink = ({ params }:{params: Object}) => {
```

Comment:
The job id in the task `storeParams` is only present _after_ the task has finished. I believe we also want to link to any currently running td job, while the task is running. To do that we can inspect the task `stateParams` which will look like this while it's running a td job:

```yaml
job:
  domainKey: 6d649edd-05b8-45f4-8ffb-aebc6883576f
  jobId: '85201301'
  pollIteration: 2
```